demisto_sdk/common/configuration.py

```diff
@@ -3,12 +3,14 @@
 
 
 class Configuration:
-    def __init__(self, default_env_dir=None, log_verbose=False, logging_level=logging.INFO):
+    def __init__(self, env_dir=None, log_verbose=False, logging_level=logging.INFO):
         logging.basicConfig(level=logging_level)
         self.log_verbose = log_verbose
-        if not default_env_dir:
-            self.env_dir = os.path.dirname(os.path.abspath(__file__))
+        self.sdk_env_dir = os.path.dirname(os.path.abspath(__file__))
+        if not env_dir:
+            self.env_dir = self.sdk_env_dir
         else:
-            self.env_dir = default_env_dir
-        self.envs_dirs_base = '{}/dev_envs/default_python'.format(self.env_dir)
+            self.env_dir = env_dir
+        self.envs_dirs_base = '{}/dev_envs/default_python'.format(self.sdk_env_dir)
```

Comment:
shouldn't it be `.format(self.env_dir)`?

Reply:
I don't think so because no normal human being will have those files in their repository

Reply:
What about cyborgs tho?